src/main/java/org/killbill/billing/entitlement/api/EntitlementApi.java

```diff
@@ -59,6 +59,22 @@ public Entitlement createBaseEntitlement(UUID accountId, PlanPhaseSpecifier spec
             throws EntitlementApiException;
 
     /**
+     * Create a new base entitlement and addOn entitlements for that account.
+     * <p/>
```

Comment:
Ah, indeed :cry: 

Let's do it in another PR. I just ran `mvn javadoc:javadoc` and there are a lot of other issues besides this (such as missing descriptions).